### PR TITLE
Add triggers to the protocol.

### PR DESCRIPTION
**Pull request by koriakin**
Protocol changes only - no implementation yet.

### CODE REVIEW

File: python/veles/async_conn/tracer.py [hunk 1]
```diff
@@ -121,6 +121,18 @@ def get_bindata_size(self, id, key):
...
+        res = node.triggers.get(key)
+        self.checks.append(check.CheckTrigger(
+            node=node.id,
+            key=key,
+            state=res,
+        ))
+        return res
+
+    def get_trigger(self, id, key):
+        return self._get_from_node(id, lambda node: self._get_trigger(node, key))
```
Comment (altran01) [inline python/veles/async_conn/tracer.py]: pep8 fails :*(

Comment (koriakin) [inline python/veles/async_conn/tracer.py]: Fixed.

**Comment by altran01** [comment]
`TriggerState` isn't generated in C++ (in PR #159 I'm changing code generation to improve it a bit - including no need to explicitly specify EnumModels to generate).

**Comment by koriakin** [comment]
Fixed.

**Pull request merged**